rmgpy/data/kinetics/family.py

```diff
@@ -1219,13 +1219,30 @@ def applyRecipe(self, reactantStructures, forward=True, unique=True):
                     for i in range(4,highest+1):
                         atomLabels['*{0:d}'.format(i)].label = '*{0:d}'.format(4+highest-i)
                         
-            elif label == 'H_shift_cyclopentadiene':
+            elif label == 'intra_ene_reaction':
                 # Labels for nodes are swapped
                 atomLabels['*1'].label = '*2'
                 atomLabels['*2'].label = '*1'
                 atomLabels['*3'].label = '*5'
                 atomLabels['*5'].label = '*3'
 
+            elif label == '6_membered_central_C-C_shift':
```

Comment:
This label and the 1,2_shiftC label need to be all lowercase, since `label.lower()` is called higher up.